composeApp/build.gradle.kts

```diff
@@ -0,0 +1,80 @@
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+
+plugins {
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.androidApplication)
+    alias(libs.plugins.jetbrainsCompose)
+}
+
+kotlin {
+    androidTarget {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "11"
+            }
+        }
+    }
+    
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
```

Comment:
We can remove iosX64 and iosSimulatorArm64 targets from both the library and the demo apps. iOS Simulator is not capable of accessing the Mac's Bluetooth, and I believe iosx64 is a target for intel based Mac's ios simulation.

Reply:
I need iOS simulator to see if the components are rendered correctly even though its bluetooth functionality does not work (I don't have an access to an iphone). But we can remove iosx64